src/pages/review.tsx

```diff
@@ -0,0 +1,138 @@
+import styled from "@emotion/styled";
+import type { NextPage } from "next";
+import Head from "next/head";
+import Image from "next/image";
+import StarFilled from "../../public/assets/star_filled.png";
+import StarOutlined from "../../public/assets/star_outlined.png";
+
+const ReviewPage: NextPage = () => {
+  return (
+    <div>
+      <Head>
+        <title>리뷰 작성</title>
+        <meta name="description" content="Generated by create next app" />
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+      <Cont>
+        <Title>마이 페이지</Title>
+        <ReviewCont>
+          <ReviewTitle>리뷰작성</ReviewTitle>
+          <Content>
+            <ImgContent>
+              <ImgCont></ImgCont>
+              <AddImgBtn>이미지 추가</AddImgBtn>
+            </ImgContent>
+            <UserReviewCont>
+              <Text>별점 및 리뷰</Text>
+              <StarCont>
```

Comment:
p3 :  축약보다는 명시적으로 적어주는 것이 좋을 것 같습니다. 
```suggestion
              <StarConunt> or <StarRating>
```

Reply:
<StarCount>말씀하시는거 맞죠? 저것은 StarContainer라고 적은건데 저번 피드백
반영해서 구체적으로 적겠습니다

Reply:
넵! 반영해주시면 재리뷰 하겟습니다